### ISSUE TITLE
MF-API Server update and integration

### ISSUE THREAD

**Issue by ogi-ts-shimizu** [ogi-ts-shimizu/pygeoapi-ogi-mf-api#1]
**Is your feature request related to a problem? Please describe.**
[MF-API Server](https://github.com/aistairc/mf-api) is an open-source Python server implementation of the [OGC API – MovingFeatures](https://ogcapi.ogc.org/movingfeatures/) with [pygeoapi](https://github.com/geopython/pygeoapi) and [MobilityDB](https://github.com/MobilityDB/MobilityDB).
The above implementation will be integrated into the [pygeoapi-mf-api](https://github.com/aistairc/pygeoapi-mf-api) and merged into the pygeoapi main branch via PR once everything is done.

**Describe the solution you'd like**
The summary of the OGC API – MovingFeatures – Part 1: Core is described in the below table. Implemented to provide the following APIs

| URL Path                                                        | Supported HTTP(s) Methods |
|-----------------------------------------------------------------|---------------------------|
| /collections/{c_id}                                             | GET,DELETE,PUT            |
| /collections/{c_id}/items                                       | GET,POST                  |
| /collections/{c_id}/items/{mf_id}                               | GET,DELETE                |
| /collections/{c_id}/items/{mf_id}/tgsequence                    | GET,POST                  |
| /collections/{c_id}/items/{mf_id}/tgsequence/{tg_id}            | DELETE                    |
| /collections/{c_id}/items/{mf_id}/tgsequence/{tg_id}/{tg_query} | GET                       |
| /collections/{c_id}/items/{mf_id}/tproperties                   | GET,POST                  |
| /collections/{c_id}/items/{mf_id}/tproperties/{tp_name}         | GET,POST,DELETE           |
| /collections/{c_id}/items/{mf_id}/tproperties/{tp_name}/{tv_id} | DELETE                    |

created and updated the following so that MF-API Server will work with the latest versions of pygeoapi(0.18) and mobilityDB(1.2.0).
- Database scheme(DDL)
- API file
- Provider file
- Test file
- docs file
- `__init__.py, flask_app.py`

**Describe alternatives you've considered**

**Additional context**
The code will only be pushed if all tests by flake-8 and pytest are successful.

**Comment by ogi-ts-shimizu** [comment]
Close as it has been merged into the Main branch.
